docs/index.md

```diff
@@ -16,12 +16,16 @@ For more of the rationale behind Autolab, please check out <a href="https://auto
  -->
 ##Components
 
-Autolab consists of two services: (1) the Ruby on Rails frontend, and (2) [Tango](/tango), the RESTful Python autograding server. <b>Either service can run independently without the other</b>. But in order to use all features of Autolab, we highly recommend installing both services.
+Autolab consists of two services: (1) the Autolab frontend which is implemented using Ruby on Rails, and (2) Tango, the RESTful Python autograding server. <b>Either service can run independently without the other.</b> But in order to use all features of Autolab, we highly recommend installing both services.
 
-While the Ruby on Rails frontend supports Autolab's web application framework, the backend Tango is responsible for distributing and completing autograding jobs and runs in virtual machines or containers. When Tango is done running a job, it then sends the autograded result back to the frontend. See below for a visualization of this system.
+While the Autolab frontend supports Autolab's web application framework, the backend Tango is responsible for distributing and completing autograding jobs, which run in virtual machines or containers. We currently support Docker and AWS virtual machines. When Tango is done running a job, it then sends the autograded result back to the frontend. Below is a visualization of the typical workflow of the Autolab system.
```

Comment:
nit: Maybe "We currently support Docker and AWS virtual machines" can go in parentheses after the previous sentence? i.e. "...which run in virtual machines or containers (we currently support Docker and AWS virtual machines)."